### PR TITLE
feat(fs): add followSymlink to expandGlob

### PR DESCRIPTION
**Pull request by lino-levan**
closes https://github.com/denoland/deno_std/issues/3000

Seems like there hasn't been a lot of discussion on this. I think this feature make sense, but if anyone has any counters I would be interested in hearing it.

### CODE REVIEW

**Comment by lino-levan** [comment]
@kt3k any opinions?

**Review by kt3k — COMMENTED** [review]
Sorry for the delay in review. This makes sense to me 👍 Can you add a test case for this?

**Comment by lino-levan** [comment]
Will do.

**Comment by chmac** [comment]
@lino-levan I'm currently stumped by this issue, and just found your PR. Thanks for creating it!

Is there anything I can do to support you in adding a test case?

**Comment by lino-levan** [comment]
Will update this PR. Not entirely sure how to test it but I think I could probably just symlink to a folder?

File: fs/expand_glob_test.ts [hunk 1]
```diff
@@ -61,7 +61,10 @@ Deno.test("expandGlobWildcard", async function () {
 
 Deno.test("expandGlobTrailingSeparator", async function () {
   const options = EG_OPTIONS;
-  assertEquals(await expandGlobArray("*/", options), ["a[b]c", "subdir"]);
+  assertEquals(await expandGlobArray("*/", options), [
+    "a[b]c",
+    "subdir",
+  ]);
```
Comment (chmac) [inline fs/expand_glob_test.ts:67]: I guess this is just a formatting change, maybe cleaner to omit this from the PR so it's easier to review?

**Comment by chmac** [comment]
> Will update this PR. Not entirely sure how to test it but I think I could probably just symlink to a folder?

I had absolutely no idea, but now I see the test cases from your example, looks good to me, but that doesn't say much! Thanks for updating this so quickly, I appreciate it.

**Comment by lino-levan** [comment]
@kt3k please take a look!

**Review by kt3k — APPROVED** [review]
LGTM. Thanks! @lino-levan

**Pull request merged**